bndtools.manual/tutorial.md

```diff
@@ -22,10 +22,20 @@ Installing Bndtools
 
 Please refer to the [Installation Instructions](installation.html).
 
+Create a Workspace Project (cnf)
+================================
+
+First we need to create a a Workspace Project. It will be named "cnf".
+
+1. From the File menu, select **New -&gt; Other -&gt; Bndtools -&gt; Bnd OSGi Workspace**.
+1. Next >
+1. Choose **GitHub -&gt; bndtools/workspace** (not osgi/workspace)
```

Comment:
Why should you not choose osgi/workspace? Or any other workspace? I think for a tutorial, it is enough to ay what you should do and not all the things you should not do :-)